pkg/blob/controllerserver.go

```diff
@@ -367,9 +367,15 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		mc.ObserveOperationWithResult(isOperationSucceeded, VolumeID, volumeID)
 	}()
 
+	secrets := req.GetSecrets()
+
+	volumeContentSourceSASToken, volumeContentSourcePath, err := d.validateVolumeContentSource(ctx, req, storageEndpointSuffix, secrets, secretName, secretNamespace, getLatestAccountKey)
```

Comment:
pls only call `validateVolumeContentSource` in the `if req.GetVolumeContentSource() != nil { ..}` logic, that would more clear logic